content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -414,31 +414,41 @@ In the CLI, the access modes are abbreviated to:
 * ROX - ReadOnlyMany
 * RWX - ReadWriteMany
 
+#### ReadWriteOncePod
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+In Kubernetes v1.22 a fourth access mode was introduced:
```

Comment:
Maybe note here that it's only supported for CSI volume types.

Reply:
Done